### PR TITLE
Create new DTP document

### PR DESCRIPTION
**Pull request by metmajer**

### CODE REVIEW

File: src/org/ods/usecase/LeVADocumentUseCase.groovy [hunk 1]
```diff
@@ -248,17 +248,23 @@ class LeVADocumentUseCase extends DocGenUseCase {
...
                 tests: this.project.getAutomatedTestsTypeUnit().collectEntries { testIssue ->
                     [
                         testIssue.key,
                         [
                             key: testIssue.key,
                             description: testIssue.description ?: "",
-                            // TODO: change template from isRelatedTo to systemRequirement
-                            systemRequirement: testIssue.requirements.collect{ it.key }.join(", ")
+                            systemRequirement: testIssue.requirements ? testIssue.requirements.join(", ") : "N/A",
+                            // TODO: should we only link in tech specs where the software design is defined?
```
Comment (metmajer) [inline src/org/ods/usecase/LeVADocumentUseCase.groovy]: @clemensutschig here, it is unclear, if we can point to any linked tech spec, or if we should only consider those that have a non-empty value set in Jira?

Comment (clemensutschig) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: @metmajer  - non empty set for requirements? ... that should not happen in the first place.

Comment (clemensutschig) [inline src/org/ods/usecase/LeVADocumentUseCase.groovy]: @metmajer  - software design is only applicable for GAMP5 .. here you need to check with QA what they want,...

Comment (metmajer) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: @clemensutschig resolved, I meant to refer to `softwareDesignSpec` instead.

Comment (metmajer) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: @clemensutschig ~so then the implementation should be good enough, as we'll issue a `N/A` if none is available~ so then we'll only reference a techSpec here if it contains a non-empty `softwareDesignSpec`.

Comment (metmajer) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: Resolved and now considers only those techSpecs that contain a softwareDesignSpec != null

**Comment by metmajer** [comment]
Screenshot 2020-03-03 at 16 15 44

**Pull request merged**